airbyte-integrations/connectors/source-close-com/Dockerfile

```diff
@@ -12,5 +12,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.0
+LABEL io.airbyte.version=0.1.1
```

Comment:
I think a minor version should be bumped, not patch

Reply:
Nothing changed at all from functional perspective. Moreover we are using the same code as we used before, but just run it in another way. Should we really bump it?

Reply:
but it does not look like a bugfix as well :)

Reply:
Agreed